packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -5,15 +5,16 @@ import { useCurrentUser } from "../common/withUser";
 import { useEventListener } from "../hooks/useEventListener";
 import { gql, useMutation } from "@apollo/client";
 import { useMulti } from "@/lib/crud/withMulti";
-import { AnalyticsContext } from "@/lib/analyticsEvents";
+import { AnalyticsContext, useTracking } from "@/lib/analyticsEvents";
 import { useLoginPopoverContext } from "../hooks/useLoginPopoverContext";
 import { useCurrentForumEvent } from "../hooks/useCurrentForumEvent";
 import range from "lodash/range";
 import sortBy from "lodash/sortBy";
 import DeferRender from "../common/DeferRender";
+import type { ForumEventVoteDisplay } from "./ForumEventResultIcon";
 
 export const POLL_MAX_WIDTH = 800;
-const SLIDER_MAX_WIDTH = 1110;
+const SLIDER_MAX_WIDTH = 1120;
```

Comment:
Tweaked to avoid icons overflowing the container and being truncated (I had already tried to address this but it still happened a bit on smaller screens)